airflow/providers/alibaba/__init__.py

```diff
@@ -29,10 +29,7 @@
 
 __version__ = "2.8.0"
 
-try:
-    from airflow import __version__ as airflow_version
-except ImportError:
-    from airflow.version import version as airflow_version
+from airflow import __version__ as airflow_version
```

Comment:
Another comment.

I think we should switch to "standard" way of getting airflow version. The `__version__` is non-standard (PEP about it has actually been rejected https://peps.python.org/pep-0396/) 

I think we've been discussing it with @Taragolis before and https://docs.python.org/3/library/importlib.metadata.html is a better/more standard way:

```python
from importlib.metadata import version  
airflow_version = version('apache-airlfow')  
```

https://docs.python.org/3/library/importlib.metadata.html

Reply:
As of Python 3.8 of course - but we are already 3.8, so ....

Reply:
Good call, thanks. Note I didn't import `version` as I thought having a str `__version__` and a method `version` that comes from importlib felt a bit weird.